mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -670,11 +680,17 @@ func flattenAlertConfigurationThresholdConfig(m *matlas.Threshold) []interface{}
 	return []interface{}{}
 }
 
-func expandAlertConfigurationNotification(d *schema.ResourceData) []matlas.Notification {
+func expandAlertConfigurationNotification(d *schema.ResourceData) ([]matlas.Notification, error) {
 	notifications := make([]matlas.Notification, len(d.Get("notification").([]interface{})))
 
 	for i, value := range d.Get("notification").([]interface{}) {
 		v := value.(map[string]interface{})
+		if v1, ok := v["interval_min"]; ok && v1.(int) > 0 {
+			typeName := v["type_name"].(string)
+			if strings.EqualFold(typeName, "pager_duty") || strings.EqualFold(typeName, "ops_genie") || strings.EqualFold(typeName, "victor_ops") {
```

Comment:
can this validation be [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/a2bbeb5f136fa1bcc25714e46f094e937a504dfa/mongodbatlas/resource_mongodbatlas_alert_configuration.go#L270) instead of an if?

Reply:
I don't think if it's possible, since I need to check another parameter `inverval_min` and if it's using or not, then validate if it's possible to use or not for certain type names(pager duty, ops genie and victor ops)